source/components/conversation/ResultReliability.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import InfoBulle from 'Ui/InfoBulle'
+
+export default function ResultReliability({ progress }) {
+	return (
+		<span>
+			<small>
+				Fiabilité du résultat :{' '}
```

Comment:
```suggestion
				Précision du résultat :{' '}
```

Avec "fiabilité" j'ai l'impression que c'est notre moteur qui n'est pas fiable et non la requête qui n'est pas assez précise.